cluster-autoscaler/config/crd/provisioning.x-k8s.io_provisioningrequests.yaml

```diff
@@ -3,11 +3,10 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    api-approved.kubernetes.io: https://github.com/kubernetes/autoscaler/pull/5848
     controller-gen.kubebuilder.io/version: v0.13.0
-  name: provisioningrequests.provisioning.k8s.io
+  name: provisioningrequests.provisioning.x-k8s.io
 spec:
-  group: provisioning.k8s.io
+  group: provisioning.x-k8s.io
```

Comment:
provisioning is pretty generic... if this will only contain autoscaling-owned APIs, consider making a more focused group name

Reply:
Do you have any naming suggestions (assuming that nothing more gets into the group in the near future)?

Reply:
single-resource API groups are sort of a smell, as is repeating the concept in the type name (ProvisioningRequest) and the group name (provisioning...).

if this is owned by sig-autoscaling, `autoscaling.x-k8s.io` doesn't seem bad

Reply:
Renamed to `autoscaling.x-k8s.io`